web-calculator.js

```diff
@@ -0,0 +1,25 @@
+var a, b, result;
```

Comment:
You should never have variables outside of a function, functions should not share state like that. If a function needs something it should be passed in as a parameter and if it needs to provide something back it should return that something.